pvr.hts/resources/language/resource.language.de_de/strings.po

```diff
@@ -60,6 +60,14 @@ msgctxt "#30010"
 msgid "Timer settings"
 msgstr "Timer"
 
+msgctxt "#30011"
+msgid "Use HTTPS"
+msgstr "Use HTTPS"
```

Comment:
This is definitely no German translation. Where does this string come from?

Reply:
Reason is that the en_GB strings.po is not correct. We need to fix that.

Reply:
Yeah, it needs to be blank here: https://github.com/kodi-pvr/pvr.hts/blob/Matrix/pvr.hts/resources/language/resource.language.en_gb/strings.po#L67